fastestimator/trace/io/model_eval.py

```diff
@@ -0,0 +1,446 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import inspect
+import json
+import locale
+import os
+import re
+import shutil
+from datetime import datetime
+from time import time
+from typing import Callable, List, Union
+
+import numpy as np
+import pydot
+from pylatex import Command, Document, Figure, Hyperref, Itemize, Label, LongTable, Marker, MultiColumn, NoEscape, \
+    Package, Section, Subsection, Subsubsection, Table, Tabular, Tabularx, TextColor, escape_latex
+from pylatex.utils import bold, dumps_list
+
+import fastestimator as fe
+from fastestimator.summary.logs.log_plot import visualize_logs
+from fastestimator.trace.trace import Trace
+from fastestimator.util.data import Data
+from fastestimator.util.latex_util import PyContainer, TabularCell, WrapText
+from fastestimator.util.traceability_util import get_environment, traceable
+from fastestimator.util.util import to_list, to_number
+
+
+@traceable()
+class TestCase():
+    """This class defines the test case that TestReport trace will take to perform auto-testing.
+
+    Args:
+        description: A test description.
+        criteria: Function to perform the test that return True when test passes and False when test fails. Input
```

Comment:
that return**s** True when **the** test passes

Reply:
fixed